python/xorbits/_mars/services/subtask/worker/processor.py

```diff
@@ -341,13 +388,40 @@ async def _store_data(self, chunk_graph: ChunkGraph):
                 storage_level,
             )
             if puts:
+                try:
+                    runner_storage: RunnerStorageActor = await mo.actor_ref(
+                        uid=RunnerStorageActor.gen_uid(self._band[1], self._slot_id),
+                        address=self._supervisor_address, # 这个supervisor_address是不是actor对应的address？
+                    )
+                except mo.ActorNotExist:
+                    # logger.debug(
+                    #     f"Can not find runner storage actor with band name `{self._band}` and slot id `{self._slot_id}",
+                    # )
+                    # self.result.status = SubtaskStatus.errored
+                    # raise
+                    runner_storage: RunnerStorageActor = await mo.create_actor(
```

Comment:
Why need this creation? The `RunnerStorageActor` is already created by `SubtaskRunnerManagerActor`.

Reply:
There used to be a 'RunnerStorage not found' error during debugging. Now the implementation is changed to raising an exception when runner storage is not found. Thank you.